dashboard/package.json

```
@@ -15,7 +15,8 @@
     "coverage": "npm run coverage:unit && npm run coverage:integration",
     "coverage:unit": "npm run test:unit && codecov",
     "coverage:integration": "npm run test:integration && codecov",
-    "dev": "nodemon ./server/index.js"
+    "dev": "nodemon ./server/index.js",
+    "generate-docs": "jsdoc -c ../jsdoc.conf.json -d ../out/dashboard/docs"
```

Let's use the full name for the options. It'll make it easier to figure out what they mean without looking at the documentation.

We can also skip the `-d` option since we'll output to the `out/` directory only.
```suggestion
    "generate-docs": "jsdoc --configure ../jsdoc.conf.json"
```

---

Done.